### PR TITLE
Completed static-pages and commited the task 45013

### PR DESCRIPTION
**Pull request by manazirahsan**

### CODE REVIEW

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (chuanhtuan) [inline app/assets/javascripts/static_pages.coffee]: remove unused file

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (chuanhtuan) [inline app/helpers/static_pages_helper.rb]: remove unused file

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "FELS", home_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",   home_path %></li>
```
Comment (chuanhtuan) [inline app/views/layouts/_header.html.erb]: 2 space

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<footer class="footer">
+  <small>
+    <%= link_to('Farmgia Bangladesh Limited', 'https://www.facebook.com/FramgiaBangladesh/?fref=ts', target: :blank) %>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   about_path %></li>
```
Comment (chuanhtuan) [inline app/views/layouts/_footer.html.erb]: 2 space

File: app/assets/stylesheets/static_pages.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (chuanhtuan) [inline app/assets/stylesheets/static_pages.scss]: remove unused file

File: app/assets/javascripts/static_pages.coffee (hunk 1 above)
Comment (manazirahsan) [inline app/assets/javascripts/static_pages.coffee]: Done with thanks.

File: app/helpers/static_pages_helper.rb (hunk 2 above)
Comment (manazirahsan) [inline app/helpers/static_pages_helper.rb]: Done.

File: app/views/layouts/_header.html.erb (hunk 3 above)
Comment (manazirahsan) [inline app/views/layouts/_header.html.erb]: Extra space removed.

File: app/views/layouts/_footer.html.erb (hunk 4 above)
Comment (manazirahsan) [inline app/views/layouts/_footer.html.erb]: Extra space removed.

File: app/assets/stylesheets/static_pages.scss (hunk 5 above)
Comment (manazirahsan) [inline app/assets/stylesheets/static_pages.scss]: Unused file removed.

**Pull request merged**